src/dodal/plans/load_eiger_metadata_plan.py

```diff
@@ -0,0 +1,144 @@
+import time
+
+import bluesky.plan_stubs as bps
+from ophyd_async.core import DetectorTrigger
+from ophyd_async.fastcs.eiger import EigerDetector, EigerTriggerInfo
+
+from dodal.devices.detector import DetectorParams
+from dodal.log import LOGGER
+
+
+def load_metadata(
+    eiger: EigerDetector,
+    enable: bool,
```

Comment:
Should: This is ROI enable, right? I think that's already in `DetectorParams.use_roi_mode`